src/it/java/com/google/checkstyle/test/chapter4formatting/rule4822variabledistance/VariableDeclarationUsageDistanceTest.java

```diff
@@ -43,6 +43,7 @@ public void testArrayTypeStyle() throws Exception {
             "219:9: " + getCheckMessage(clazz, msgExt, "t", 5, 3),
             "483:9: " + getCheckMessage(clazz, msgExt, "myOption", 7, 3),
             "495:9: " + getCheckMessage(clazz, msgExt, "myOption", 6, 3),
+            "508:9: " + getCheckMessage(clazz, msgExt, "count", 4, 3),
         };
```

Comment:
it may be obvious. but this added violation because we are now making the check to do a violation to move the variable near the block our check didn't make this violation before the same was applied for all other input files